scripts/sanity_chk/sanitylib.py

```diff
@@ -3229,6 +3232,45 @@ def csv_report(self, filename):
                     rowdict["rom_size"] = rom_size
                 cw.writerow(rowdict)
 
+    def set_default(self, obj):
+        ''' JSON doesn't recognize set type so need to converted to list '''
+        if isinstance(obj,set):
+            return list (obj)
+        raise TypeError
+
+    def json_report(self, filename):
+        rowdict = {}
+        rowdict["environment"] = {"OS": os.name,
```

Comment:
captialized here, later all keys are small cap.

Reply:
changed it to be consistent